gradle/libs.versions.toml

```diff
@@ -5,6 +5,7 @@ greenmail = '2.0.1'
 groovy = '4.0.23'
 gsp = '7.0.0-SNAPSHOT'
 javamail = '2.0.1'
+servlet-api = '6.0.0'
```

Comment:
should this be taken from the bom?

Reply:
It doesn't look like we have the servlet-api in the bom, but perhaps it should be there?  Just because it is in the bom, doesn't me it will be resolved.

Reply:
Perhaps the other versions, could be resolved from the bom?  gsp, groovy, etc?